src/Nethermind/Nethermind.Wallet/ProtectedKeyStoreWallet.cs

```diff
@@ -78,8 +80,7 @@ public bool UnlockAccount(Address address, SecureString passphrase, TimeSpan? ti
                 if (result.ResultType == ResultType.Success)
                 {
                     if (_logger.IsInfo) _logger.Info($"Unlocking account: {address}");
-                    _unlockedAccounts.Add(key.Address.ToString(), _protectedPrivateKeyFactory.Create(key),
-                        new CacheItemPolicy() {Priority = CacheItemPriority.NotRemovable, AbsoluteExpiration = _timestamper.UtcNowOffset + (timeSpan ?? DefaultExpirationTime)});
+                    _unlockedAccounts.Set(key.Address.ToString(), _protectedPrivateKeyFactory.Create(key));
```

Comment:
What about expiration here (based on `timeSpan` or `DefaultExpirationTime`)?

Reply:
i dont think LRU supports expiry. Should we implement it?

Reply:
That's a great question for @dceleda or @LukaszRozmej who may understand the purpose of this wallet. I don't know even how much data is cached here. No expire sounds scary but on the other hand, it may be completely meanigless.

Reply:
LGTM 😅